Vagrantfile

```diff
@@ -1,14 +1,16 @@
 Vagrant.configure(2) do |config|
+  config.vm.synced_folder ".", "/mk"
   config.vm.define "trusty64" do |trusty64|
     trusty64.vm.box = "ubuntu/trusty64"
     trusty64.vm.provider "virtualbox" do |v|
       v.memory = 2048
     end
     trusty64.vm.provision "shell", inline: <<-SHELL
+      sudo apt-add-repository -y ppa:maxmind/ppa
```

Comment:
This is not necessary anymore. In master we have now removed libmaxminddb. I will revert this specific hunk when I am going to merge the diff in master.